src/components/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -13,26 +13,30 @@
       width="auto"
       @opened="state.hidePopover = false"
     >
-      <KButton
-        appearance="tertiary"
-        aria-role="input"
-        class="timepicker-input"
-        :class="{ 'set-min-width': hasTimePeriods }"
-        data-testid="k-datetime-picker-input"
-        :style="widthStyle"
-      >
-        <CalIcon
-          v-if="icon"
-          class="calendar-icon"
-          :color="KUI_COLOR_TEXT_NEUTRAL"
-          :size="KUI_ICON_SIZE_20"
-        />
-        <div
-          class="timepicker-display"
-          data-testid="k-datetime-picker-display"
+      <div class="datetime-picker-input-wrapper">
+        <span
+          class="datetime-picker-display"
+          :class="{ 'has-icon': icon, 'disabled': disabled }"
+          data-testid="datetime-picker-display"
           v-html="state.abbreviatedDisplay"
         />
-      </KButton>
+
```

Comment:
@mihai-peteu switched from `KButton` to `KInput read-only` here as it's gonna be much easier to maintain similarity styling-wise

Reply:
This is a great solution. No more awkward forced styling to make it look like a Kinput...

Going to do a quick test on khcp-ui and make sure everything looks good.